docs/permissions/plugin-authors/04-authorizing-access-to-paginated-data.md

```diff
@@ -15,7 +15,7 @@ One possible solution may leverage the batching functionality to authorize all o
 -     res.json(getAll())
 +     const items = getAll();
 +     const decisions = await permissions.authorize(
-+       items.map(({ id }) => ({ permission: todosListRead, resourceRef: id })),
++       items.map(({ id }) => ({ permission: todoListReadPermission, resourceRef: id })),
```

Comment:
This permission was named inconsistently, standardised on this name which follows the convention established for the other permissions.